kymatio/scattering2d/frontend/torch_f.py

```diff
@@ -0,0 +1,178 @@
+__all__ = ['Scattering2D']
+
+import torch
+from ..backend import cdgmm, Modulus, SubsampleFourier, fft, Pad, unpad, NAME
```

Comment:
Why not import the whole backend as a module? It gets a little confusing here since `NAME` now refers to the name of the backend used, not the frontend.

Reply:
Hmm that would work, it requires a little more work, and also, it sort of "hardcodes" what is a backend.(right now, we could evolve freely to what you propose or something simpler)

@lostanlen @eickenberg any opinions?

Reply:
> it sort of "hardcodes" what is a backend.

I don't understand?

At the very least, we should do `from ..backend import NAME as BACKEND_NAME`.

Reply:
I've switched it to "BACKEND_NAME". What I meant is that if the backend is a NN module, it can be restrictive... I think this could be changed in a future PR?